babel.config.js

```diff
@@ -1,4 +1,12 @@
 module.exports = {
   presets: [ 'module:metro-react-native-babel-preset' ],
   plugins: [ [ '@babel/plugin-proposal-decorators', { legacy: true } ] ],
+  env: {
+    production: {
+      plugins: [ 'transform-remove-console' ],
```

Comment:
We should probably have ESLint catch any console statements

Reply:
I want console logs in dev just so in flipper I can see what is happening. We should have a good standardized logger but for now this does the trick. https://reactnative.dev/docs/performance#using-consolelog-statements